test/java/seedu/addressbook/logic/ExamBookTest.java

```diff
@@ -0,0 +1,4 @@
+package seedu.addressbook.logic;
+
+public class ExamBookTest {
+}
```

Comment:
Just a stub?

Reply:
oops i created this but forgot to delete